ProjectCode/src/HouseObjects/Room.java

```diff
@@ -1,9 +1,25 @@
 package HouseObjects;
 
+import java.util.ArrayList;
+
 /**
 *
 * @author a_richard
 */
+//room contains Arraylist of windows and doors
 public class Room {
+	ArrayList<Window> window = new ArrayList<Window>();
```

Comment:
These were added in a previous commit, so we won't need to add it again. I'll also review your algorithm to see if the method names match the ones already there